sunpy/map/tests/test_mapbase.py

```diff
@@ -1555,6 +1556,36 @@ def test_rotation_rect_pixelated_data(aia171_test_map):
     rect_rot_map = rect_map.rotate(30 * u.deg)
     rect_rot_map.peek()
 
+@figure_test
+def test_draw_contours_with_transform():
+    # TODO: Use test data
+    from sunpy.data.sample import AIA_171_IMAGE, HMI_LOS_IMAGE
```

Comment:
I think we would need to solve this?

Reply:
Is there a reason to not use a fixture here?

Reply:
I can turn it into a fixture, but should we be using the sample data for this? 

I can try with the test aia image and see how well it works first?

Reply:
Oh we lack a hmi test file. 

I will just turn them into a fixture

Reply:
I setup two fixtures instead.